package/toltec-bootstrap/package

```diff
@@ -5,7 +5,7 @@
 pkgnames=(toltec-bootstrap)
 pkgdesc="Manage your Toltec install"
 url=https://toltec-dev.org/
-pkgver=0.4.1-1
+pkgver=0.4.2-1
```

Comment:
Not sure if the -1 should go

Reply:
pkgrel is required, from https://github.com/toltec-dev/toltec/blob/stable/docs/package.md#pkgver-field
> Always include a dash-separated package revision number at the end of the version, resetting it to 1 when bumping the software version, and increasing it when making changes to the recipe itself